functions/venue.js

```diff
@@ -163,7 +163,7 @@ const createVenueData_v2 = (data, context) => ({
   },
   owners: [context.auth.token.user_id],
   showGrid: data.showGrid || false,
-  columns: data.columns || 1,
+  ...(showGrid && { columns: data.columns }),
```

Comment:
should we set `rows` as well?

Reply:
Rows are automatically calculated in the Map.tsx (Party map component)